velox/dwio/common/BufferedInput.h

```diff
@@ -122,6 +122,14 @@ class BufferedInput {
     return std::make_unique<BufferedInput>(input_, pool_);
   }
 
+  std::unique_ptr<SeekableInputStream> readFile(
```

Comment:
I don't think API fits the BufferedInput class. This function is used only once anyway.
Let's inline this code in the call site.